automl/beta/src/main/java/com/example/automl/TablesGetModel.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.automl;
+
+// [START automl_tables_get_model]
+
+import com.google.cloud.automl.v1beta1.AutoMlClient;
+import com.google.cloud.automl.v1beta1.Model;
+import com.google.cloud.automl.v1beta1.ModelName;
+import com.google.cloud.automl.v1beta1.TablesModelColumnInfo;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+
+public class TablesGetModel {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "YOUR_PROJECT_ID";
+    String region = "YOUR_REGION";
+    String modelId = "YOUR_MODEL_ID";
+    getModel(projectId, region, modelId);
+  }
+
+  // Demonstrates using the AutoML client to get model details.
+  public static void getModel(String projectId, String computeRegion, String modelId)
+      throws IOException {
```

Comment:
Shouldn't this also include the more specific exceptions beside `IOException`, like `io.grpc. StatusRuntimeException`, `io.grpc.StatusException`, and `com.google.protobuf.InvalidProtocolBufferException` so that users can diagnose what's really going on?

Reply:
Yes, almost forgot that.